stdlib/regex.gr

```diff
@@ -611,135 +665,253 @@ parseClass = (buf: RegExBuf) => {
     }
   }
 },
-
 parsePosixCharClass = (buf: RegExBuf) => {
-  if (!more(buf)) {
+  if (!(more(buf))) {
     Err(parseErr(buf, "Missing POSIX character class after `[`", 0))
   } else {
-    match(peek(buf)) {
+    match (peek(buf)) {
       Err(e) => Err(e),
       Ok(':') => {
         ignore(eat(buf, ':'))
-        let rec loop = (acc) => {
-          match(peek(buf)) {
+        let rec loop = acc => {
+          match (peek(buf)) {
             Err(e) => Err(e),
             Ok(':') => {
               ignore(eat(buf, ':'))
-              match(eat(buf, ']')) {
+              match (eat(buf, ']')) {
                 Err(_) => Err(parseErr(buf, "Missing closing `]`", 0)),
-                Ok(_) => Ok(List.join("", List.reverse(acc)))
+                Ok(_) => Ok(List.join("", List.reverse(acc))),
               }
             },
-            Ok(c) when (Char.code('a') <= Char.code(c) && Char.code(c) <= Char.code('z')) => {
+            Ok(c) when Char.code('a') <= Char.code(c) && Char.code(
+              c,
+            ) <= Char.code('z') => {
```

Comment:
This feels unfortunate! Could the breaks maybe happen if the infix operands can't fit on the same line?
```
Ok(c) when Char.code('a') <= Char.code(c) &&
Char.code(c) <= Char.code('z')
```

Reply:
Fixed in next push